dev

```diff
@@ -326,6 +326,11 @@ cmd_dev() {
   fi
 }
 
+cmd_backdate() {
+  docker-compose exec django /docker_venv/bin/python3 ./codewof/codewof_utils.py backdate_points_and_badges
+}
+defhelp backdate 'Re-calculates points and badges earned for all user profiles.'
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398747-12088480-de0d-11e9-9347-ff777890803c.png)

I get an error attempting this, but might be using it wrong